lib/events/adapters/aws_sns.ex

```diff
@@ -36,18 +37,22 @@ defmodule ElixirTools.Events.Adapters.AwsSns do
   @spec add_envelope(Event.t(), [publish_opt]) :: map
   defp add_envelope(event, opts) do
     uuid_module = opts[:uuid_module] || UUID
+    timex_module = opts[:timex_module] || Timex
 
     config = Application.get_env(:pagantis_elixir_tools, ElixirTools.Events)[:adapter_config]
     group = opts[:group] || Map.fetch!(config, :group)
 
     uuid_seed_2 = "#{event.name}-#{event.version}-#{event.event_id_seed_optional}"
     id = uuid_module.uuid5(event.event_id_seed, uuid_seed_2)
 
+    occurred_at = event.occurred_at || timex_module.now()
+    occurred_at = Timex.format!(occurred_at, "{ISO:Extended:Z}")
```

Comment:
instead of double assigning it, wdyt about moving this to line 55? That also makes it more consistent with line 53 where we transform the value, like we do for `occured_at`